data/korath/korath outfits.txt

```diff
@@ -178,6 +204,241 @@ outfit "Control Transceiver"
 	description "This is a miniaturized communication device, which can be connected to all the major systems of a Kor Sestor automaton, apparently as a way of controlling them remotely. The communications are encrypted and authenticated in a way that would make it nearly impossible to spoof the control signals."
 
 
+outfit "Afterburner (Asteroid Class)"
+	plural "Afterburners (Asteroid Class)"
+	category "Engines"
+	"cost" 136000
+	thumbnail "outfit/tiny korath afterburner"
+	"mass" 10
+	"outfit space" -10
```

Comment:
```suggestion
	"mass" 9
	"outfit space" -9
```

The other suggestion for changes here became outdated, so creating a new suggestion for it. This merely reduces the mass and outfit space by 1, which improves the Korath utility-density slightly, and gives it a different use profile from the human afterburner at size 10. As in there may be some ships that can't fit that afterburner, but could fit this one. And having such diversity improves the game.